src/main/java/it/pagopa/interop/probing/response/updater/dto/ChangeResponseReceived.java

```diff
@@ -0,0 +1,29 @@
+package it.pagopa.interop.probing.response.updater.dto;
+
+import java.time.OffsetDateTime;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Accessors(chain = true, fluent = true)
+public class ChangeResponseReceived {
```

Comment:
DTO need to be named with postfix Dto and it need to implement an interface. Please get in touch with Alessandro for this issue.